### ISSUE TITLE
Unknown config options

### ISSUE THREAD

**Issue by KevinBurton** [neo4j/docker-neo4j#120]
I am starting the lateest neo4j docker image from DockerHub. When I start the container I get some warnings:
```
neo4j_1  | 2018-01-25 03:30:34.780+0000 WARN  Unknown config option: causal_clustering.transaction_listen_addres
neo4j_1  | 2018-01-25 03:30:34.766+0000 WARN  Unknown config option: causal_clustering.discovery_listen_address
neo4j_1  | 2018-01-25 03:30:34.776+0000 WARN  Unknown config option: causal_clustering.raft_advertised_address
neo4j_1  | 2018-01-25 03:30:34.776+0000 WARN  Unknown config option: causal_clustering.raft_listen_address
neo4j_1  | 2018-01-25 03:30:34.777+0000 WARN  Unknown config option: ha.host.coordination
neo4j_1  | 2018-01-25 03:30:34.777+0000 WARN  Unknown config option: causal_clustering.transaction_advertised_address
```

I have a volume bound for the database but I don't know where the configuration file is. It is probably part of the image. The relevant section of the docker-compose.yml file looks like:

```
  neo4j:
    image: neo4j
    ports:
        - "7474:7474"
        - "7687"
    volumes:
      - type: bind
        source: /Users/rebeccaannburton/Projects/AspNetCore-Angular/neo4j/data/db
        target: /data
```

Ideas?

Kevin

**Comment by Robbie1977** [comment]
Related to https://github.com/neo4j/docker-neo4j/pull/83

**Comment by nguyenhuuloc304** [comment]
I have the same issue today. I cannot find neo4j.conf in my host volumn 
![image](https://user-images.githubusercontent.com/21157325/51098637-582d3300-17fe-11e9-9e81-3b9a19f797b9.png)

Anyone can help me this issue?

**Comment by nguyenhuuloc304** [comment]
@KevinBurton did you resolve this problem? Could you give me advises?

**Comment by jennyowen** [comment]
TLDR: You don't need to worry about these warnings, you can safely ignore them.

It's because we use the same `docker-entrypoint.sh` for both Neo4j community and enterprise editions. The configuration options it warns about are to do with clustering, which is only a feature of the enterprise edition. In the `docker-entrypoint.sh` we need to set a bunch of clustering configuration options which are only relevant to enterprise edition. Since you are running the community edition, those configuration options aren't required, so it gives a warning about them being there.

You could definitely argue that we should only be setting those options in the enterprise Neo4j docker container, and not in the community container. As it stands, there's no easy way of telling what edition of Neo4j has been installed, only the version number. Adding such a check would require some new command line fixes to core Neo4j. Again you could (correctly in my opinion!) argue that we should do that. Unfortunately this is quite a big change, and since the Neo4j Community container still works despite these annoying warnings, this is a low priority issue. Sorry about that.
